### ISSUE TITLE
508-defect-2: Focus on alert header after showing submit error

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#55334]
### Point of contact

Josh Kim

### Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

### Details

After submitting an application but an error is thrown, focus should be sent to the alert's `H3`

<img width="671" alt="review & submit page showing a error alert with focus on header of you're decision review request didn't go through" src="https://user-images.githubusercontent.com/136959/225957659-90d56064-f3ed-418d-8ba1-f9e462131c20.png">

### Reproduction steps

1. I had to duplicate this issue locally by changing the API endpoint...
2. Get to the review & submit page and submit

### Proposed solution or next steps

The submission error component is provided to the form config, so within it we can shift focus to the `H3` once the alert is visible

### References, articles, or WCAG support

1.
2.
3.
...

### Type of issue

- [X] Screenreader
- [ ] Keyboard
- [X] Focus
- [ ] Headings
- [ ] Color
- [ ] Zoom
- [ ] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by joshkimux** [comment]
@Mottie couldn't manage to replicate getting the error 😢 

https://user-images.githubusercontent.com/14154792/226364156-ad70ec99-a0bf-4f7f-93c4-681d0b2b0227.mov

There's larger issues on this review answers page which I could use your help figuring out what's in/vs out of our scope/responsibilities.

**Comment by sortizsh** [comment]
Josh said we could close this out based on Robin's testing in local environment
